Irc.ClassExtensions/Irc.Helpers.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

[nitpick]: Is it not better to do a `Directory.Build.props` file in the same directory as the solution file `Irc7.sln` with:

```props
<Project>
  <PropertyGroup>
    <LangVersion>13.0</LangVersion>
    <TargetFrameworkVersion>net9.0</TargetFrameworkVersion>
    <Prefer32Bit>false</Prefer32Bit>
  </PropertyGroup>
</Project>
```

And then in all the projects you can do:

```suggestion
    <TargetFramework>$(TargetFrameworkVersion)</TargetFramework>
```

This way you only have to set the version in one location.

---

hey you're absolutely welcome to do that change